src/PaylineProvider.tsx

```diff
@@ -33,7 +33,10 @@ const PaylineProvider: React.ComponentType<PropsType> = ({ production = false, c
 
     script.addEventListener('load', onLoad, false);
     return () => {
-      if (script) script.removeEventListener('load', onLoad, false);
+      if (script) {
+        script.removeEventListener('load', onLoad, false);
+        document.head.removeChild(script)
```

Comment:
Removing the script tag won't remove the attached `window.Payline` object. Therefore I prefer not to clean the DOM